### DISCUSSION TITLE
How to refresh the table every X seconds.

### DISCUSSION THREAD

**Discussion by EG-Mohamed** [Laravel-Backpack/community-forum#542] [General] [answered]
Hello everyone, how to refresh the table every X seconds?

**Comment by promatik** [comment, marked as answer]
Hi @EGYWEB-Mohamed!
I believe you are talking about the entity list table — datatables (https://datatables.net/)

If so, there is a method on `window.crud.table` that you can you use; `ajax.reload()`, just wrap it inside a `setInternal` 👌

```js
setInterval(window.crud.table.ajax.reload, 10e3) // 10 secs
```

**Reply by EG-Mohamed** [reply]
Hi @promatik , Yes exactly I already did this but on the main list view so it's the best practice, or should I use the script Widget

**Reply by promatik** [reply]
Hi @EGYWEB-Mohamed sorry, I didn't understand...
Are you asking where to place that JS? If so, yes, a script widget is probably the best option 👌

**Reply by EG-Mohamed** [reply]
Yes exactly that what i already did by add it to script Widget
